test/tools/ossfuzz/SolCustomMutator.h

```diff
@@ -0,0 +1,43 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include <test/tools/ossfuzz/SolidityLexer.h>
+#include <test/tools/ossfuzz/SolidityMutator.h>
+#include <test/tools/ossfuzz/SolidityParser.h>
+
+#include <cstddef>
+#include <cstdint>
+#include <ostream>
+#include <random>
+
+namespace solidity::test::fuzzer
+{
+struct SolCustomMutator
```

Comment:
I think this class needs a more self-explanatory name. When I started reading this code I was wondering why it just calls the other mutator without doing anything extra. Now I see that one is a mutator and the other is just part of its implementation.

Reply:
Sorry for the confusion. May be `SolMutatorInterface` is more appropriate here?

Reply:
Changed name.

Reply:
It's much better because it does the make the distinction clear. It now sounds like an abstract base class serving as an interface though. Maybe `SolidityCustomMutatorWrapper`?

Or you could rename both clases to something like:
- `SolidityCustomMutatorWrapper` -> `SoliditySourceCodeMutator`
- `SolidityMutator` -> `SolidityASTMutator`